### DISCUSSION TITLE
Allow linebreak in bullet points with SHIFT+ENTER

### DISCUSSION THREAD

**Discussion by hlascelles** [outline/rich-markdown-editor#361] [Ideas] [answered]
When writing a list of bullet points, it would be nice to match Word (and various other tools) "SHIFT+ENTER" to line break without making a new bullet point.

ie end with:

* Some Text
  And this
* And now a new bullet point
  

I know the same can be achieved in this editor using ENTER then subsequently BACKSPACE, but... Since SHIFT+ENTER doesn't appear to do anything else, can we make it do the same thing for those with the muscle memory?

NB, I am not suggesting CTRL+ENTER as for many apps that means "submit" or similar.

**Comment by tommoor** [comment, marked as answer]
Seems like a good idea to me! Shame discussions can't be converted into issues

**Reply by tommoor** [reply]
Resolved in https://github.com/outline/rich-markdown-editor/commit/96e50285087dffcdd4d99f9a7941174fc7ccdc73
